src/Ocean/SplitExplicit01/Communication.jl

```diff
@@ -11,8 +11,11 @@ using Printf
     dgFast,
     Qfast,
     S_fast,
-    slow_dt, rkC, rkW,
-    s, nStages,
+    slow_dt,
+    rkC,
+    rkW,
+    s,
+    nStages,
```

Comment:
Happy to approve this. I do have a question about style. Most of the code is written using snakecase, but here some variables appear in camelcase. Should we pick one style and stick with it?

Reply:
We should fix this in a separate PR to make sure @jm-c is OK with it and to keep PRs clean.